apps/flame-defi/app/pool/context/pool-position-context-v2.tsx

```diff
@@ -2,11 +2,20 @@
 
 import { useParams } from "next/navigation";
 import { createContext, PropsWithChildren, useState } from "react";
+import type { Address } from "viem";
+
+import { TransactionStatus } from "@repo/flame-types";
 
 export interface PoolPositionContextProps {
   tokenId: string;
```

Comment:
Nit: I know this is old, but I think it got renamed at one point from `positionId` or maybe `nftId`. I know you're just matching the term uniswap uses but it confuses me almost every time for a split second asking myself which token it's referring to then realizing it's the non fungible token id lol. I think `positionId` for use in our interfaces is a lot easier to grok. Wdyt?